app/assets/javascripts/discourse/app/models/category.js

```diff
@@ -310,16 +310,6 @@ const Category = RestModel.extend({
     }
   },
 
-  @discourseComputed("id", "topicTrackingState.messageCount")
-  unreadTopics(id) {
-    return this.topicTrackingState.countUnread({ categoryId: id });
-  },
-
-  @discourseComputed("id", "topicTrackingState.messageCount")
-  newTopics(id) {
-    return this.topicTrackingState.countNew({ categoryId: id });
-  },
```

Comment:
We can remove the computed property here but I think it is still useful to keep the logic of how to lookup the topic tracking state in the model as a method.